python/py-Pyro5/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-Pyro5
+version             5.15
+revision            0
+license             MIT
+maintainers         nomaintainer
+description         Distributed object middleware for Python (RPC)
+long_description    \
+    Pyro means PYthon Remote Objects. It is a library that \
+    enables you to build applications in which objects can talk to each \
+    other over the network, with minimal programming effort. You can just \
+    use normal Python method calls, with almost every possible parameter \
+    and return value type, and Pyro takes care of locating the right object \
+    on the right computer to execute the method. It is designed to be very easy \
+    to use, and to generally stay out of your way. But it also provides a set \
+    of powerful features that enables you to build distributed applications \
+    rapidly and effortlessly. Pyro is written in 100% pure Python.
+
+homepage            https://github.com/irmen/Pyro5
+platforms           {darwin any}
+supported_archs     noarch
+
+checksums           rmd160  2fb49334b4038f6959cd7ffca87977b8cb0e1d7c \
+                    sha256  82c3dfc9860b49f897b28ff24fe6716c841672c600af8fe40d0e3a7fac9a3f5e \
+                    size    447559
+
+python.versions     39 310 311
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
this shouldn't be needed anymore because of the default `python.pep517 yes` - correct?

Reply:
Let me try